bip-schnorr.mediawiki

```diff
@@ -91,9 +91,7 @@ This proposal suggests to include the tag by prefixing the hashed data with ''SH
 
 === Specification ===
 
-We first describe the key generation algorithm, then the verification algorithm, and then the signature algorithm.
-
-The following convention is used, with constants as defined for secp256k1:
+The following conventions are used, with constants as defined for [http://www.secg.org/sec2-v2.pdf secp256k1]:
```

Comment:
Can we change this to HTTPS? (Also the first time we link to it.)

ACK otherwise

Reply:
Done.